components/Group/controllers/GroupController.ts

```diff
@@ -0,0 +1,84 @@
+import { GroupService } from '../services/GroupService';
+import { Request, Response } from 'express';
+import { UserService } from '../../User/services/UserService';
+
+
+export default class GroupController {
+	public static async getAllGroups(req: Request, res: Response) {
+		try {
+			const groups = await GroupService.getAllGroups();
+			return res.status(200).json(groups);
+		} catch(err) {
+			res.status(500).json({ error: err.message });
+		}
+	}
+
+	public static async getGroupById(req: Request, res: Response) {
+		const { id } = req.params;
+		try {
+			const group = await GroupService.getGroupById(id);
+			group
+				? res.status(200).json(group)
+				: res.status(500).send(`Group with ${id} not found :(`);
```

Comment:
Here and in other controllers:
Better to use 404 when some resource has not been found. 500 - for server error.